src/Inkluzitron/Extensions/GraphicsExtensions.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Drawing;
+
+namespace Inkluzitron.Extensions
+{
+    static public class GraphicsExtensions
+    {
+        private const string Ellipsis = "...";
+
+        static public SizeF ShrinkAndMeasureString(this Graphics g, ref string text, Font font, float maxWidth, bool appendEllipsis = false)
```

Comment:
Tady bych to obrátil, protože se mi jeví, že primární účel je změřit text a side effect je, že se ořeže. Tudíž by se to mělo jmenovat `MeasureAndShrinkText`.